packages/components/src/style.scss

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal Dependencies
+ */
+@import 'animation-slider/style.scss';
```

Comment:
Is there a reason to do this?  I like having the css directly included via the component.

Reply:
The package build process doesn't extract CSS from JS (it doesn't use webpack, just babel), so we [build the CSS separately and compile it with node-sass](https://github.com/woocommerce/wc-admin/blob/update/packages-components/bin/packages/build.js#L118).

Reply:
Thanks for mentioning it, I haven't seen that before. Right now we don't have a need to keep the css imports in the component files (since we're using this extra scss "entry point" file), but it might be useful in the future 👍